### PR TITLE
ClaraDopler making it better

### PR DESCRIPTION
**Pull request by ClaraDopler**

### CODE REVIEW

File: testing/test_app.py [hunk 1]
```diff
@@ -4,5 +4,8 @@
 from pytest_mqtt import __version__
 
 
+
+# just a random comment
```
Comment (DelilahMichaels) [inline testing/test_app.py:8]: Do we really need this?

File: testing/test_util.py [hunk 2]
```diff
@@ -7,3 +7,7 @@ def test_probe_tcp_connect_available():
 
 def test_probe_tcp_connect_unavailable():
     assert probe_tcp_connect("9.9.9.9", 444) is False
+
+
+def test_probe_tcp_connect_unavailable():
+    assert probe_tcp_connect("1.2.3.4", 444) is True
```
Comment (DelilahMichaels) [inline testing/test_util.py:13]: This should be 5.6.7.8

**Review by DelilahMichaels — CHANGES_REQUESTED** [review]
You should fix these things!

**Review by BethanyJ123 — APPROVED** [review]
I don't care what ClaraDopler says, it looks fine to me

File: testing/test_app.py (hunk 1 above)
Comment (BethanyJ123) [reply testing/test_app.py:8]: It's fine

File: testing/test_util.py (hunk 2 above)
Comment (BethanyJ123) [reply testing/test_util.py:13]: I think 1.2.3.4 will work

**Review by DelilahMichaels — APPROVED** [review]
I guess that is OK.

**Pull request merged**